eng/scripts/Test-Packages.ps1

```diff
@@ -58,11 +58,21 @@ foreach ($package in $packagesToTest) {
 
     Invoke-LoggedCommand "cargo build --keep-going" -GroupOutput
     Write-Host "`n`n"
+    # test individual targets so we can exclude the bench target which in some cases requires
+    #deployed resources
+    Invoke-LoggedCommand "cargo test --lib --no-fail-fast" -GroupOutput
+    Write-Host "`n`n"
 
-    Invoke-LoggedCommand "cargo test --doc --no-fail-fast" -GroupOutput
+    Invoke-LoggedCommand "cargo test --bins --no-fail-fast" -GroupOutput
+    Write-Host "`n`n"
+
+    Invoke-LoggedCommand "cargo test --examples --no-fail-fast" -GroupOutput
     Write-Host "`n`n"
 
-    Invoke-LoggedCommand "cargo test --all-targets --no-fail-fast" -GroupOutput
+    Invoke-LoggedCommand "cargo test --tests --no-fail-fast" -GroupOutput
+    Write-Host "`n`n"
+
+    Invoke-LoggedCommand "cargo test --doc --no-fail-fast" -GroupOutput
```

Comment:
Most options can be combined. See `cargo test --help`. IIRC, you could do `cargo test --lib --tests --bins --examples` followed by a separate `cargo test --doc` because `--doc` can't be combined with the rest...even though it is by default (never understood that).

Reply:
done